docker-images/kafka-connect/s2i/Dockerfile

```diff
@@ -14,5 +14,4 @@ COPY ./scripts/ $S2I_HOME
 
 USER 1001:1001
 
-ENTRYPOINT [ ]
 CMD /opt/s2i/run
```

Comment:
Question (my ignorance) : Why not using the same syntax as in the other CMD ? I mean CMD ["/opt/s2i/run"]

Reply:
This was leftover from before. Should be fixed now.